dozer/cogs/roles.py

```diff
@@ -357,16 +369,18 @@ def from_role(cls, role):
 
 
 class MissingMember(db.DatabaseObject):
+    """Is this necessary any more?"""
     __tablename__ = 'missing_members'
     guild_id = db.Column(db.Integer, primary_key=True)
     member_id = db.Column(db.Integer, primary_key=True)
     missing_roles = db.relationship('MissingRole', back_populates='member', cascade='all, delete, delete-orphan')
 
 
 class MissingRole(db.DatabaseObject):
+    """Actually, what does this do?"""
     __tablename__ = 'missing_roles'
     __table_args__ = (
-    db.ForeignKeyConstraint(['guild_id', 'member_id'], ['missing_members.guild_id', 'missing_members.member_id']),)
+        db.ForeignKeyConstraint(['guild_id', 'member_id'], ['missing_members.guild_id', 'missing_members.member_id']))
```

Comment:
The trailing comma is necessary to make this a tuple with one value, rather than extra parentheses.